liquibase-core/src/main/java/liquibase/serializer/core/xml/XMLChangeLogSerializer.java

```diff
@@ -197,7 +197,8 @@ private void setValueOnNode(Element node, String objectNamespace, String objectN
         } else if (value instanceof Map) {
             for (Map.Entry entry : (Set<Map.Entry>) ((Map) value).entrySet()) {
                 Element mapNode = currentChangeLogFileDOM.createElementNS(LiquibaseSerializable.STANDARD_CHANGELOG_NAMESPACE, qualifyName(objectName, objectNamespace, parentNamespace));
-                setValueOnNode(mapNode, objectNamespace, (String) entry.getKey(), entry.getValue(), serializationType, objectNamespace);
+                setValueOnNode(mapNode, objectNamespace, "name", entry.getKey(), serializationType, objectNamespace);
+                setValueOnNode(mapNode, objectNamespace, "value", entry.getValue(), serializationType, objectNamespace);
```

Comment:
There's some funny namespace handling in there (originally), I'm not entirely sure 1) it is all correct and 2) my change produces expected results in all cases.  The change works as expected for the `customPrecondition` and `customChange` at least, which are the only uses I've found so far.